### PR TITLE
uncommented

### PR DESCRIPTION
**Pull request by Jeena01**
ok to merge

### CODE REVIEW

**Comment by Jeena01** [comment]
fixed the issue

File: rosen_backend/siteConfig/settings.py [hunk 1]
```diff
@@ -76,20 +76,20 @@
...
-        'NAME': BASE_DIR / 'db.sqlite3',
-    }
-}
-
 # DATABASES = {
 #     'default': {
-#         'ENGINE': 'djongo',
-#         'NAME': 'rosenDB',
+#         'ENGINE': 'django.db.backends.sqlite3',
+#         'NAME': BASE_DIR / 'db.sqlite3',
```
Comment (yuqi88) [inline rosen_backend/siteConfig/settings.py:82]: Shouldn't this be the part that is uncommented for now?

Comment (aliL2000) [reply rosen_backend/siteConfig/settings.py:82]: Yeah do not merge this, this has to be uncommented

Comment (yuqi88) [reply rosen_backend/siteConfig/settings.py:82]: Okay, I'll closes this PR, so nobody accidentally close

**Comment by Jeena01** [comment]
closing

**Pull request closed without merging**